cypress/integration/html-reader/navigations.ts

```diff
@@ -9,80 +9,91 @@ describe('navigating an EPUB page', () => {
     cy.findByRole('link', { name: 'Return to Homepage' }).should(
       'have.prop',
       'href',
-      '/'
+      `${Cypress.config().baseUrl}/`
     );
   });
 
   it('should update page content after clicking on TOC link', () => {
-    cy.iframe(IFRAME_SELECTOR)
-      .findByRole('img', {
-        name: "Alice's Adventures in Wonderland, by Lewis Carroll",
-      })
-      .should('exist');
+    cy.intercept('GET', 'https://alice.dita.digital/text/chapter-1.xhtml').as(
+      'chapterOne'
+    );
 
-    cy.iframe(IFRAME_SELECTOR)
-      .findByText('Down the Rab­bit-Hole')
-      .should('not.exist');
+    cy.getIframeBody(IFRAME_SELECTOR)
+      .find('img', { timeout: 10000 })
+      .should(
+        'have.attr',
+        'alt',
+        "Alice's Adventures in Wonderland, by Lewis Carroll"
+      );
 
-    // Open TOC menu
+    cy.log('open TOC menu');
     cy.findByRole('button', { name: 'Table of Contents' }).click();
 
-    // Open chapter 1
+    cy.log('open chapter 1');
     cy.findByRole('menuitem', { name: 'I: Down the Rab­bit-Hole' }).click();
 
-    cy.log('briefly see the loading indicator');
-    cy.get('#reader-loading').should('be.visible');
-    cy.get('#reader-loading').should('not.be.visible');
+    cy.wait('@chapterOne', { timeout: 10000 }).then((interception) => {
+      assert.isNotNull(
+        interception?.response?.body,
+        'chapter one API call has data'
+      );
+    });
+
+    cy.wait(3000);
```

Comment:
Even after adding the intercepts, I still have to add these wait() commands. This is because the following command is running before the iframe has fully updated, so it is flaky. In CPW I mitigated this by checking the iframe src attribute, but that same attribute doesn't exist in the new web reader. So I think wait() is our best bet for now.